R/onLoad.R

```diff
@@ -3,7 +3,8 @@
   op.rJavaEnv <- list(
     # this default folder choice is in-line with what renv package does
     # https://github.com/rstudio/renv/blob/d6bced36afa0ad56719ca78be6773e9b4bbb078f/R/bootstrap.R#L940-L950
-    rJavaEnv.cache_path = tools::R_user_dir("rJavaEnv", which = "cache")
+    rJavaEnv.cache_path = tools::R_user_dir("rJavaEnv", which = "cache"),
+    rJavaEnv.valid_major_java_versions = c("8", "11", "17", "21", "22")
```

Comment:
It feels a bit surprising to me to put this in an option? Under what circumstances would the user want to change it?

Reply:
Well, they just might. Also, this allows me to set the cache folder to temp directory in examples, as I was worried about CRAN review and "nowhere except temp dir" policy.